README.md

```diff
@@ -71,6 +71,10 @@ TLA+ is a formal specification and verification language to help engineers desig
 * [Practical TLA+](https://www.apress.com/gp/book/9781484238288)
 * [TLA+ in Practice and Theory](https://pron.github.io/posts/tlaplus_part1) (freely available)
 
+### Parsers
+* [tree-sitter-tlaplus](https://github.com/tlaplus-community/tree-sitter-tlaplus): A tree-sitter grammar for TLA⁺ and PlusCal 
```

Comment:
SANY's and TLAPS' parser should probably be listed at the top.

Reply:
Sounds fair! I've just added them, not sure if maybe you have better links